.github/workflows/reviewdog.yml

```diff
@@ -0,0 +1,24 @@
+name: reviewdog
+on: [pull_request]
+jobs:
+  reviewdog:
+    name: reviewdog
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
```

Comment:
this is the only occurrence of Go v1.17. Everywhere else, you're using 1.16

Reply:
fixed